Assets/MixedRealityToolkit.SDK/Features/Utilities/PointerUtils.cs

```diff
@@ -144,6 +144,159 @@ public static IEnumerable<IMixedRealityPointer> GetPointers()
                 }
             }
         }
-    }
 
+        /// <summary>
+        /// Gets the physics ray based on the interaction enabled pointer assosiciated with the head gaze.
+        /// </summary>
+        /// <param name="ray">The physics ray for the pointer</param>
+        /// <returns>
+        /// True if the ray contains valid data, false otherwise.
```

Comment:
When would the head ray contain invalid data? I'm not sure this needs to be a try get. Isn't getting the head gaze always just grabbing Camera.Main.position and Camera.Main.Rotation?

Note: even if the gaze pointer is not active, I would expect this method to still return the direction that the head is pointing. Same for the hands -- if a hand exists in the scene, even if the line pointer is not active, I would expect the method to return a direction.

Reply:
replaced with a GetHeadGazeRay that creates the ray based on the Camera transform's position and forward